flake.nix

```diff
@@ -2,14 +2,9 @@
   description = "Chainweb";
 
   inputs = {
-    nixpkgs.url = "github:NixOS/nixpkgs?rev=4d2b37a84fad1091b9de401eb450aae66f1a741e";
```

Comment:
why?

i would much rather us pin our own nixpkgs. we have our own cache so we don't need to depend on IOG for that.